src/libOpenImageIO/imagebuf.cpp

```diff
@@ -1418,8 +1418,20 @@ ImageBuf::write(ImageOutput* out, ProgressCallback progress_callback,
             int chunk = clamp(round_to_multiple(int(budget / slsize), 64), 1,
                               1024);
             std::unique_ptr<char[]> tmp(new char[chunk * slsize]);
+
+            const bool isDecreasingY = !strcmp(out->format_name(), "openexr")
```

Comment:
I fully admit this is not great. I am totally open to suggestions on the preferred way to hide this behind the ImageOutput abstraction. I wasn't sure if a new virtual method was in order or if this should be a new "feature name" passed to supports().